pydotorg/tests/test_context_processors.py

```diff
@@ -24,3 +25,9 @@ def test_url_name(self):
 
         mock_request = MockRequest(path='/nothing/here/')
         self.assertEqual({}, context_processors.url_name(mock_request))
+
+        mock_request = MockRequest(path='/about/')
+        self.assertEqual({'BLOG_URL': settings.PYTHON_BLOG_URL}, context_processors.blog_url(mock_request))
+
+        mock_request = MockRequest(path='/')
```

Comment:
We can remove this one. I know you added it to test `IndexView` case, but since we are unit testing the context processor itself, we can safely remove the `path='/'` test.